index.html

```diff
@@ -21,33 +21,122 @@ <h2 class="notoc">Contents</h2>
 
 
 
-
-
+<section id="when_should_review_be_requested">
+  <h2>When should wide review be requested?</h2>
+  <p>A document is available for review from the moment it is first <a
+  href='https://www.w3.org/Consortium/Process/#publishing'>published</a>.
+  Working Groups should <a
+  href='https://www.w3.org/Consortium/Process/#formally-addressed'>formally
+  address</a> any substantive review comment about a technical report in a
+  timely manner.</p>
+  <p>Wide review should or must be requested:</p>
+  <ul>
+  <li>After a <a href="https://www.w3.org/Consortium/Process/#first-wd">First
+  Public Working Draft</a> is published (for most documents).
+  <p>Working Groups are often reluctant to make substantive changes to a mature
+  document, so reviewers should get a chance to send substantive technical
+  reviews as early as possible. This is especially important for <a
+  href="#how_to_get_horizontal_review">horizontal reviews</a>.</p>
+  </li>
+  <li>Before a document gets advanced to <a
+  href="https://www.w3.org/Consortium/Process/#transition-cr">Candidate
+  Recommendation</a> or gets updated as a <a
+  href="https://www.w3.org/Consortium/Process/#update-requests">Candidate
+  Recommendation Snapshot</a>.
+  <p>For those, the <a href="https://www.w3.org/Consortium/Process/">W3C
+  Process</a> requires a Group to show that the specification has received wide
+  review.</p>
+  </li>
+  <li>When a document is both reasonably stable and still flexible enough to
+  allow changes where issues are identified.</li>
+  <li>When new features are added after a document has already received wide
+  review (perhaps requesting a review limited to the new features).</li>
+  </ul>
+  </section>
 
 <section id="who_to_ask_for_review">
-<h2>Who to ask for review?</h2>
- 
-<p>Much of this document focuses on how and when to conduct horizontal reviews, but they are only a subset of a full wide review, which must also include other stakeholders including Web developers, technology providers and implementers not active in the Working Group, external groups and standards organizations working on related areas, etc. Here is a list of suggestions:</p>
+<h2>Who to ask for wide review?</h2>
+
+ <p>The objective is to ensure that the entire set of stakeholders of the Web
+ community, including the general public, have  adequate notice of the progress
+ of the Working Group and are genuinely able to perform reviews of and provide
+ comments on the specification. When considering <a
+ href="https://www.w3.org/Guide/transitions">requests to advance the maturity
+ level of the document</a>, the Director will consider who has been explicitly
+ offered a reasonable opportunity to review the document.</p>
+
+<p>You <strong>must</strong> publish an updated technical report, with the Status of the Document
+indicating clearly that you're looking for <em>wide review</em>, before inviting
```

Comment:
All of our publications are inviting review, that is why we publish them. Let's not add more junk to the SOTD.

Reply:
Except that the other groups need a clearer signal that wide review is expected before moving to CR. It's not junk, it gets picked up by our tools for the past few years.